blocks/library/latest-posts/index.php

```diff
@@ -56,6 +56,10 @@ function render_block_core_latest_posts( $attributes ) {
 		$class .= ' columns-' . $attributes['columns'];
 	}
 
+	if ( isset( $attributes['className'] ) && ! empty( $attributes['className'] ) ) {
```

Comment:
Using `empty()` means that `isset()` isn't needed. This could be simplied to just:

`if ( ! empty( $attributes['className'] ) ) {`.

Reply:
Nice catch and it encouraged me to take a bit deeper look at this. 
ATM `$attributes['className']` can return NULL, an empty string or a string that was populated by a user. 

Considering that "0" is a valid string that theoretically can be used by a user as a custom class `empty()` won't be enough as it will validate it to false. 

Also as it's a user input where you can expect unexpected I should have been sanitized it. ie if I put `<script>alert(1)</script>` as a custom class I get an alert prompt on the front-end. Shame on me!

To sum up, I think that the conditional should have a similar shape to what you can see below, but holding off for now because of the other comments.

```php
$custom_class = wp_strip_all_tags( $attributes['className'] );
if ( isset( $custom_class ) && $custom_class !== '' ) {
	$class .= ' ' . $custom_class;
}
```